docker-bake.hcl

```diff
@@ -327,71 +327,53 @@ target "osbuild-ci-latest" {
         )
 }
 
+variable "CENTOS_PACKAGES" {
+    default = <<EOF
+bash
+bubblewrap
+coreutils
+cryptsetup
+curl
+dnf
+dnf-plugins-core
+dosfstools
+e2fsprogs
+findutils
+git
+glibc
+iproute
+lvm2
+make
+ostree
+policycoreutils
+python-rpm-macros
+python3
+python3-devel
+python3-iniparse
+python3-jsonschema
+python3-librepo
+python3-mako
+python3-pip
+python3-pyyaml
+python3-rpm-generators
+python3-rpm-macros
+qemu-img
+rpm
+rpm-build
+rpm-ostree
+rpmdevtools
+skopeo
+systemd
+systemd-container
+tar
+util-linux
+fuse-overlayfs
+EOF
+}
+
 target "virtual-osbuild-ci-cXs" {
         args = {
-                OSB_DNF_PACKAGES = join(",", [
-                        "bash",
-                        //"btrfs-progs",         // not available in cXs
-                        "bubblewrap",
-                        "coreutils",
-                        "cryptsetup",
-                        "curl",
-                        "dnf",
-                        "dnf-plugins-core",
-                        "dosfstools",
-                        "e2fsprogs",
-                        "findutils",
-                        "git",
-                        "glibc",
-                        "iproute",
-                        "lvm2",
-                        "make",
-                        //"nbd",                 // not available in cXs
-                        //"nbd-cli",             // not available in cXs
-                        "ostree",
-                        //"pacman",              // not available in cXs
-                        "policycoreutils",
-                        //"pylint",              // not available in cXs
-                        "python-rpm-macros",
-                        "python3",               // install just the default version
-                        //"python3.6",
-                        //"python3.7",
-                        //"python3.8",
-                        //"python3.9",
-                        //"python3.10",
-                        //"python3.12",
-                        //"python3-autopep8",    // not available in cXs
-                        //"python3-boto3",       // not available in cXs
-                        //"python3-botocore",    // not available in cXs
-                        //"python3-docutils",    // not available in cXs
```

Comment:
So, the intention with the commented out items was to keep the list the same with Fedora container, with the packages that are not available in cXs commented out with an explanation. Because it may be confusing to have some tools installed as an RPM package in Fedora, but using pip in centos.

If not a big deal, I would keep it that way.

Reply:
We probably can't keep comments with the new list, right? :thinking:

Reply:
Let me describe the whole story:)
For easier modification / review it is the best to keep this as a multi line string (as before). The problem is that bake syntax allows multiline string in here-doc format only. The other problem is that it was impossible in bake to append inherited variable from base image, so the only way to have this common module list was to put it as a separate variable.

There are three possible solutions, every one not perfect:
1) I will add all the comments on top of the new variable but they will no longer be inline comments
2) I could add them inline and modify dnf script processing it later to filter them out when it is processing list of packages.
3) Copy paste the list with comments to each image definition with it own comments (which kills the whole inheritance idea).

I would opt for option (1) as I agree it is worth to keep those comments